pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -189,7 +189,7 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Language and compiler for writing highly efficient custom Deep-Learning primitives";
     homepage = "https://github.com/openai/triton";
-    platforms = lib.platforms.unix;
+    platforms = platforms.linux;
```

Comment:
Changes to this file would require a secondary commit.

Reply:
Done.